### ISSUE TITLE
NordVPN: could not find VPN_HOST in the VPN configuration

### ISSUE THREAD

**Issue by SignumPL** [guillaumedsde/alpine-qbittorrent-openvpn#23]
## Information

I'm trying to use custom ovpn configuration (NordVPN provider), but container fails to start:
```Recreating qbit_qbittorrent_1 ... done
Attaching to qbit_qbittorrent_1
qbittorrent_1  | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
qbittorrent_1  | [s6-init] ensuring user provided files have correct perms...exited 0.
qbittorrent_1  | [fix-attrs.d] applying ownership & permissions fixes...
qbittorrent_1  | [fix-attrs.d] done.
qbittorrent_1  | [cont-init.d] executing container initialization scripts...
qbittorrent_1  | [cont-init.d] 01-setup-permissions: executing...
qbittorrent_1  | [cont-init.d] 01-setup-permissions: exited 0.
qbittorrent_1  | [cont-init.d] 02-setup-openvpn: executing...
qbittorrent_1  | Wed Sep  2 20:56:48 2020 TUN/TAP device tun0 opened
qbittorrent_1  | Wed Sep  2 20:56:48 2020 Persist state set to: ON
qbittorrent_1  | OpenVPN provider not set. Using configuration at /config/openvpn/config.ovpn
qbittorrent_1  | [cont-init.d] 02-setup-openvpn: exited 0.
qbittorrent_1  | [cont-init.d] 03-setup-iptables: executing...
qbittorrent_1  | awk: cmd. line:1: Is a directory
qbittorrent_1  | awk: cmd. line:1: Is a directory
qbittorrent_1  | awk: cmd. line:1: Is a directory
qbittorrent_1  | ERROR could not find VPN_HOST in the VPN configuration
qbittorrent_1  | [cont-init.d] 03-setup-iptables: exited 1.
qbittorrent_1  | [cont-finish.d] executing container finish scripts...
qbittorrent_1  | [cont-finish.d] done.
qbittorrent_1  | [s6-finish] waiting for services.
qbittorrent_1  | [s6-finish] sending all processes the TERM signal.
qbittorrent_1  | [s6-finish] sending all processes the KILL signal and exiting.
qbit_qbittorrent_1 exited with code 1
```

## Current setup

_information about your current setup_

| docker image tag: | latest    |
| -------------------------------------------------------- | --- |
| docker image hash (ex: `603b78e07727`)                   |  a45e8ae29070   |

## `docker-compose.yml` file or `docker run` command

_how did you start the container?_ (don't forget to use backticks for creating a proper code block)

`docker-compose up`

```version: "3.3"
services:
  qbittorrent:
    volumes:
      - "/mnt/storage/qbittorrent/downloads:/downloads"
      - "/mnt/storage/qbittorrent/config:/config"
      - "/mnt/storage/qbittorrent/pl122.nordvpn.com.udp1194.ovpn:/config/openvpn/config.ovpn"
      - "/etc/localtime:/etc/localtime:ro"
    environment:
      - OPENVPN_PROVIDER=
      - OPENVPN_USERNAME=***
      - OPENVPN_PASSWORD=***
      - PUID=1001
      - PGID=1001
      - LAN=192.168.1.0/24
    ports:
      - "9092:8080"
    cap_add:
      - NET_ADMIN
    image: guillaumedsde/alpine-qbittorrent-openvpn:latest
```

## Attempted Fix(es)

_What you have tried in order to fix the issue_ (if anything)

**Comment by CoreyVidal** [comment]
I am having this exact same error.

**Comment by JohnAustin16** [comment]
I had the same issue but then i maped my folder andi did not include the file and it worked. Heres my compose file.

version: '3'

services:
 Qbit-Trans:
  image: guillaumedsde/alpine-qbittorrent-openvpn:latest
  container_name: Movie-Qbit
  restart: unless-stopped
  cap_add:
   - NET_ADMIN
  ports:
   - 8080:8080
   - 9013:9013
  volumes:
   - /home/super/dockers/Qbit/config:/config
   - /tank/Data/Downloads/Movies/Qbit:/downloads
   - /etc/localtime:/etc/localtime:ro
   - /home/super/Containers/OVPN:/config/openvpn
  environment:
   - OPENVPN_USERNAME=
   - OPENVPN_PASSWORD=
   - LAN=

My host has the folder /home/super/Containers/OVPN inside this folder i have my ovpn file by the name of config.ovpn and /config/openvpn is the folder inside the container

So if you try changeing "/mnt/storage/qbittorrent/pl122.nordvpn.com.udp1194.ovpn:/config/openvpn/config.ovpn" to 
"/mnt/storage/qbittorrent:/config/openvpn"

And change your ovpn file name from pl122.nordvpn.com.udp1194.ovpn to config.ovpn on your host machine.

Once you get qbit running your /mnt/storage/qbittorrent will inherit these properties drwx------ 2 systemd-timesync systemd-journal just a FYI.

This setup works for me hopefully you get it running with this as well.

**Comment by guillaumedsde** [comment]
hi @CoreyVidal @SignumPL I've implemented a fix, see my comment here #33 let me know if that fixes it for you

**Comment by guillaumedsde** [comment]
I'll close this issue, but feel free to reopen if the fix mentionned above doesnt work :)
